packages/clerk-js/src/ui/Components.tsx

```diff
@@ -309,6 +311,12 @@ const Components = (props: ComponentsProps) => {
       setState(s => ({ ...s, [`${component}Prefetch`]: true }));
     };
 
+    componentsControls.flushEmotionCache = () => {
+      emotionCache.sheet.flush();
+      emotionCache.inserted = {};
+      emotionCache.registered = {};
+    };
```

Comment:
I got the idea [here](https://ntsim.uk/posts/how-to-update-or-remove-global-styles-in-emotion#the-aha-moment)